fastlane/Fastfile

```diff
@@ -760,66 +761,56 @@ REPOSITORY_NAME = 'WordPress-Android'
       github_release_prefix: "",
       exclusions: []
     },
-  ]
+  ].freeze
 
-  private_lane :send_strings_for_translation do |options|
-    sh("cd .. && mkdir -p #{update_strings_path} && cp #{main_strings_path} #{update_strings_path} && git add #{update_strings_path}strings.xml")
-    sh('git diff-index --quiet HEAD || git commit -m "Send strings to translation."')
-    sh('git push origin HEAD')
-  end
-
-  private_lane :commit_strings do |options|
-    if options[:auto_commit] then
-       sh("cd .. && git add #{main_strings_path}")
-       sh("git commit -m 'Update strings for translation'")
-       sh('git push origin HEAD')
-    else
-      UI.important("Your #{main_strings_path} has changed.")
-      UI.input('Please, review the changes, commit them and press return to continue.')
+  lane :update_frozen_strings_for_translation do |options|
+    Dir.chdir('..') do
+      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
+      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
     end
```

Comment:
What do you think of moving this note in the code itself? Could be something shorter, too, e.g.

```suggestion
  lane :update_frozen_strings_for_translation do |options|
    # It's necessary to `cd` to the parent directory because commands running directly from
    # the `Fastfile` run from the `./fastlane` folder
    # 
    # See: https://docs.fastlane.tools/advanced/fastlane/#directory-behavior
    Dir.chdir('..') do
      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
    end
```